docs/sdk-docs/installing-dapp/README.md

```diff
@@ -0,0 +1,40 @@
+# Install and Run the dApp
+
+Clone the Raiden Light Client repository.
+
+```bash
+git clone --recurse-submodules https://github.com/raiden-network/light-client.git
+```
+
+## Build the SDK
+
+Change to the `raiden-ts` directory, install the dependencies and build the SDK.
+
+```bash
+cd ../raiden-ts
+npm install
+npm run build
```

Comment:
I never ran `npm run build`, @kelsos is this necessary?

Reply:
It was at some point, I still run it but maybe just to be sure @taleldayekh can create a new environment and go through the steps one by one to make sure that they are needed and up to date.

Reply:
I can confirm that it works without `npm run build` :)

Reply:
I usually just do `npm install` and I'm all set for local development. I think™ `npm run build` is necessary for publishing to npm

Reply:
I would still prefer it if we can verify in a clean VM that this actually works as intended when there is no previous state left. 

I had a lot of cases where something worked on my development environment properly but started failing when someone new tried to run the dApp locally in a new environment.